app/(tabs)/classes.tsx

```diff
@@ -1,11 +1,76 @@
-import { View, Text } from "react-native";
+import React from 'react';
+import { View, ScrollView, StyleSheet } from 'react-native';
+import Colors from '../utils/Colors';
+import ClassCard from '../components/classCard';
+import { time } from 'console';
```

Comment:
you can delete this import since it's not necessary

Reply:
done